theme/settings.yml

```diff
@@ -31,7 +31,7 @@ project_categories:
       - College affordability
       - STEM
       - Support HBCU’s
-    color: cat-education
+    color: cat-foobar
```

Comment:
This is where I'm setting the new color `cat-foobar` for the category Education.